lib/package-snippets-view.js

```diff
@@ -76,11 +109,25 @@ export default class PackageSnippetsView {
     }
   }
 
-  addSnippets () {
+  getSnippetsModule () {
+    const snippetsPackage = atom.packages.getLoadedPackage('snippets')
+    return snippetsPackage ? snippetsPackage.mainModule : null
```

Comment:
You can find the service object definition [here](https://github.com/atom/snippets/blob/e9fc6eece2ad7142c070ce9e502e1a045a030e7e/lib/snippets.coffee#L420), just exposing the `getUserSnippetsPath` should be good enough!

Reply:
Changing the reference to `mainModule` in this added section is easy enough. There is preexisting code in `getSnippets` that deals with the module directly instead of the exposed service object — will that need to get addressed before we can land this, or can I disregard it for now?

Reply:
@daviwil Do you think we should open an issue for this?

https://github.com/atom/settings-view/blob/57b4f01cb2ece1815a9eb0494227daf5b735f11c/lib/package-snippets-view.js#L100

Reply:
Interesting, I didn't realize there's pre-existing code for this.  I just took a look at how we consume the snippet service and it appears we don't even pass along the object directly but instead pass along its `getSnippets` method with a different object:

https://github.com/atom/settings-view/blob/master/lib/main.js#L86-L90

It looks like this code definitely needs to be overhauled at some point in the future so I'd suggest leaving the current usage of `snippetsPackage.mainModule` the way it is for now.

If it's not too much extra work to get the user settings service method piped through, it'd be ideal if we could at least do that part well without the rest of it being fixed.  If it ends up adding more headache than it's worth, I'd be OK with just calling directly into the `mainModule` to get the config path.

Reply:
> If it's not too much extra work to get the user settings service method piped through, it'd be ideal if we could at least do that part well without the rest of it being fixed. If it ends up adding more headache than it's worth, I'd be OK with just calling directly into the mainModule to get the config path.

Nah, that part was easy enough. The rest I’d be happy to tackle in a separate issue.

Reply:
Excellent, thanks a lot!